addons/vehicles_dragoon/CfgWeapons.hpp

```diff
@@ -0,0 +1,159 @@
+class CfgWeapons {
+    class CannonCore;
+    class autocannon_Base_F: CannonCore {
+        class player;
+    };
+    class autocannon_30mm_CTWS: autocannon_Base_F
```

Comment:
We can make a cav_30mm_ctws cannon and use this for our vehicles to ensure no other mod vehicles a accedently also get effected.

Reply:
The point of this update IS to overwrite the turret by adding new magazines and ammunition. It shouldn't break the turret. I can remove the redundant changes and add the append changes but this is the best way we get it to work imo without having to modify the turret, which has caused other things to break  (such as causing sound bugs) in the past. We avoid those with this in my testing.

Reply:
I'm unsure how to update the turret without having the sound glitches return

Reply:
And this causes issues with the sound?
```
cav_autocannon_30mm_CTWS : autocannon_30mm_CTWS {}
```

Reply:
I am unsure. It may be there or in the redefinition of the Turret classes in CfgVehicles.

Reply:
I just gave this another shot. It looks like it might work given how I did the inheritance for the turret.